cmd/bucket-main.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
```

Comment:
why do we need this file? There are no `bucket` commands

Reply:
because ILM and lock are not merged yet